src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/VariantOverview.module.css

```diff
@@ -35,11 +35,11 @@
 .variantType {
   font-size: 12px;
   font-weight: var(--font-weight-light);
-  margin-left: 0.6rem;
+  margin-left: 12px;
 }
 
 .labelTextStrong {
-  font-size: 12px;
+  font-size: 13px;
```

Comment:
this is the default body font size; shouldn't need to specify it unless you are overriding some other styles